index.js

```diff
@@ -4,6 +4,9 @@ var hasFlag = require('has-flag');
 var serializeError = require('serialize-error');
 var Runner = require('./lib/runner');
 var runner = new Runner();
+var cache = require('cacha')('.ava/cache');
+
+cache.clean();
```

Comment:
@sindresorhus yes, this makes sense. I'm up to something like [`update-notifier` Configstore](https://github.com/yeoman/update-notifier/blob/master/index.js#L39).